From fba4c8062aea3d4a4c5b8a1877c5a359c94c6407 Mon Sep 17 00:00:00 2001
From: quic_zivin <quic_zivin@quicinc.com>
Date: Fri, 20 Sep 2024 13:42:32 +0800
Subject: [PATCH 1/8] QCLINUX: arm64: dts: qcom: sa8255p: Modify correct dt 
 name for SCM

This change is required as this node is looked up and
patched by hypervisor.

Change-Id: I131a9cde36b3db6ab73f713f6a94f5da42c17a33
Signed-off-by: quic_zivin <quic_zivin@quicinc.com>
---
 arch/arm64/boot/dts/qcom/sa8255p.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sa8255p.dtsi b/arch/arm64/boot/dts/qcom/sa8255p.dtsi
index 8a1a8c9f3a48..ee84dc2dae8d 100644
--- a/arch/arm64/boot/dts/qcom/sa8255p.dtsi
+++ b/arch/arm64/boot/dts/qcom/sa8255p.dtsi
@@ -212,7 +212,7 @@ core3 {
 	};
 
 	firmware: firmware {
-		scm {
+		qcom_scm {
 			compatible = "qcom,scm-sa8255p", "qcom,scm";
 			memory-region = <&tz_ffi_mem>;
 			qcom,dload-mode = <&tcsr 0x13000>;
-- 
2.34.1

